visidata/_input.py

```diff
@@ -481,16 +481,16 @@ def _drawPrompt(val):
             cur_input_key = keys[(i+offset)%len(keys)]
 
     retargs = {}
-    if record:
-        lastargs = {}
-        for k, input_kwargs in kwargs.items():
-            v = input_kwargs.get('value', '')
-            retargs[k] = v
+    lastargs = {}
+    for k, input_kwargs in kwargs.items():
+        v = input_kwargs.get('value', '')
+        retargs[k] = v
 
+        if record:
```

Comment:
This one should be `input_kwargs.get('record', record)`, so that in e.g. username/password forms, the username can be saved but the password won't be.